grp/perf.grp

```diff
@@ -35,7 +35,23 @@ local p,pos;
   else
     p:=12+p;
   fi;
-  ReadGrp(Concatenation("perf",String(p),".grp"));
+  name:=Concatenation("perf",String(p),".grp");
+  libname := SHALLOW_COPY_OBJ( "grp" );
+  APPEND_LIST_INTR( libname, "/" );
+  APPEND_LIST_INTR( libname, name );
+  while not READ_GAP_ROOT( libname ) do
+    Error("\n\n",
+    "For reasons of size, the perfect groups library for orders >10^6 is\n",
+    "not distributed fully by default. To access the group requested, get\n",
+    "the file ",name," from\n",
+    "https://github.com/hulpke/extraperfect\n",
+    "and put it in the `grp` subdirectory of your GAP installation. Then",
+    " type \n\nreturn;\n\n",
+    "to continue in this GAP session ",
+    "(which will read in the file).\n\n\n");
```

Comment:
This is indeed reasonable for now, and we can easily replace this with something "better" (whatever that'll mean) once/if we have anything.

Reply:
Yes. My first idea for "better" was NrPerfectLibraryGroups could check if the files are available. Back when the perfect groups library had holes in it, I used NrPerfectLibraryGroups to run tests or searches over the library. I agree the current message is very clear and very informative for an interactive session. I followed the instructions, used `return` and the search continued without any trouble.